crates/rover-std/src/fs.rs

```diff
@@ -258,7 +258,7 @@ impl Fs {
     {
         let path = path.as_ref().to_path_buf();
         tokio::task::spawn_blocking(move || {
-            eprintln!("watching {} for changes", path.as_std_path().display());
+            infoln!("Watching {} for changes", path.as_std_path().display());
```

Comment:
Sorry for bringing this up now. I'm not sure my brain was fully functioning last week. I just sanity checked the guidelines we we've been using in rover [here](https://clig.dev/#the-basics).
It seems like we might want to update `infoln` to print to `stderr`, as there's a guide there to use `stderr` for all messaging and `stdout` for output and this preserves the existing behavior (sans emojis)

Reply:
Ah. I see that now. I should try reading for a change.